.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: monthly
```

Could you please change the interval to weekly (In our other repos we use weekly interval)
```suggestion
      interval: weekly
```